src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -68,6 +68,9 @@ class AdapterSpec:
     # set of training instances.  Used to compute error bars.
     num_train_trials: int = 1
 
+    # Sample train examples or use deterministic
```

Comment:
Make the comment clearer: Sample train examples (as opposed to taking the first few in order)

Reply:
But just curious why the sample_train = False mode is needed?  In general, it'd be nice not to add more things into AdapterSpec unless they are necessary.

Reply:
Here, we are providing models bios in context for steerability. I disabled sampling so that I can ensure that we run through all the bios that are in-context examples, rather than just a randomly chosen subset.

Reply:
Sampling is needed so that we can go through all the training examples (which are our steering groups) rather than sampling a random subset of them